docs/validators/upgrades/v4.0-upgrade.md

```diff
@@ -2,6 +2,8 @@
 
 This document provides guidelines for node operators upgrading from Regen Ledger v3.0 to v4.0. For a general overview of the upgrade process, please refer to [Upgrade Overview](README.md).
 
+*Update: A patch release (`v4.1.0`) was released soon-after the v4.0 upgrade. This document has been updated to use the `v4.1.0` as the upgrade binary rather than `v4.0.0`.*
```

Comment:
A chain must upgrade to `v4.0` before they upgrade to `v4.1`. When I said "patch" release, I was referring to the releases with non-state breaking changes (major `v4`, minor `v4.1`, patch `v4.1.5`). We cannot instruct validators to use the `v4.1` release when attempting to update to `v4.0` at the defined upgrade height.